src/tools/sfdk/command.cpp

```diff
@@ -1930,22 +1930,35 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
     if (arguments.first() == "list") {
         QCommandLineParser parser;
         QCommandLineOption availableOption(QStringList{"available", "a"});
+        QCommandLineOption snapshotsOption(QStringList{"snapshots", "s"});
         QCommandLineOption slowOption("slow");
-        parser.addOptions({availableOption, slowOption});
+        parser.addOptions({availableOption, snapshotsOption, slowOption});
 
         if (!parser.parse(arguments)) {
             qerr() << parser.errorText() << endl;
             return BadUsage;
         }
 
+        if (!P::checkExclusiveOption(parser, {&availableOption, &snapshotsOption}))
+            return BadUsage;
+
+        if (parser.isSet(snapshotsOption) && typeHint == SdkManager::ToolingHint) {
+            qerr() << tr("The '%1' option can only be used with targets")
```

Comment:
As the `--snapshots` option can be used also without `targets`, i.e. `sfdk tools list --snapshots`, I would rephrase this as "The '%1' option can not be used with toolings".

Reply:
yeah I wish sdk-assistant was never using the strange UI with the optional [tooling|target] "subcommand", which is actually an option and so should use the natural --option syntax instead. A few things would have been much easier and clearer, this being just one example. Fixed and aligned on sdk-assistant side as well.